### PR TITLE
Add ability to restart failed build items

### PR DESCRIPTION
**Pull request by anfimovdm**

### CODE REVIEW

File: src/pages/Build.vue [hunk 1]
```diff
@@ -220,6 +228,15 @@ export default defineComponent({
         return {label: distribution.name, value: distribution.name}
       })
     },
+    failedItems () {
+      let rebuilt = true
+      for (let task of this.build.tasks) {
+        if (task.status < BuildStatus.COMPLETED) {
```
Comment (AvoidMe) [inline src/pages/Build.vue:234]: if (task.status !== BuildStatus.FAILED) {

**Review by AvoidMe — CHANGES_REQUESTED** [review]

File: src/pages/Build.vue (hunk 1 above)
Comment (anfimovdm) [reply src/pages/Build.vue]: Fixed

File: src/pages/Build.vue [hunk 2]
```diff
@@ -220,6 +228,15 @@ export default defineComponent({
         return {label: distribution.name, value: distribution.name}
       })
     },
+    failedItems () {
+      let rebuilt = true
+      for (let task of this.build.tasks) {
+        if (task.status !== BuildStatus.COMPLETED) {
```
Comment (VanessaRish) [inline src/pages/Build.vue:234]: why COMPLETED ? If we need failed to rebuild

**Review by VanessaRish — CHANGES_REQUESTED** [review]

File: src/pages/Build.vue (hunk 2 above)
Comment (anfimovdm) [reply src/pages/Build.vue]: My bad, fixed

**Review by soksanichenko — APPROVED** [review]

**Pull request merged**